### DISCUSSION TITLE
[MSYS] ld: cannot find -lintl: No such file or directory

### DISCUSSION THREAD

**Discussion by ghost** [msys2/MSYS2-packages#4338] [Q&A] [answered]
Which packages I need to install to have this library?

```
$ python-config --libs
-lpython3.11 -lintl -ldl
```

```
$ gcc -shared example.o example_wrap.o `python-config --libs` -o test.dll
/usr/lib/gcc/x86_64-pc-msys/13.2.0/../../../../x86_64-pc-msys/bin/ld: cannot find -lintl: No such file or directory
collect2: error: ld returned 1 exit status
```

```
msys/libintl 0.22.4-1 (libraries) [installed]
    GNU Internationalization runtime library
```

**Comment by Biswa96** [comment]
Are you using mingw or msys2 toolchain? What is the output of `type gcc` ?

**Reply by ghost** [reply]
This question is about the MSYS GCC.

**Reply by MehdiChinoune** [reply]
are you calling gcc from pwsh?
Open MSYS shell and replace `'python-config --libs'` with `$(python-config --libs)`

**Reply by ghost** [reply]
It gives the same error. It seems it can't find `intl` even though I have that library installed.

**Reply by ghost** [reply]
Btw, what is `pwsh`? I'm using the normal MSYS shell.

**Reply by MehdiChinoune** [reply]
pwsh: PowerShell.

**Reply by MehdiChinoune** [reply, marked as answer]
Try installing gettext-devel, It should fix your issue.
